models/CoronSegmentator/runtests.sh

```
@@ -0,0 +1,373 @@
+    echo "    -v, --version     : show MONAI and system version information and exit"
+    echo ""
+    echo "${separator}For bug reports and feature requests, please file an issue at:"
+    echo "    https://github.com/Project-MONAI/model-zoo/issues/new/choose"
+    echo ""
+    echo "To choose an alternative python executable, set the environmental variable, \"MONAI_MODEL_ZOO_PY_EXE\"."
+    exit 1
+}
+
+function check_import {
+    echo "Python: ${PY_EXE}"
+    ${cmdPrefix}${PY_EXE} -W error -W ignore::DeprecationWarning -c "import monai"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)